src/common/meta/src/key/datanode_table.rs

```diff
@@ -86,15 +86,22 @@ pub struct DatanodeTableValue {
     pub table_id: TableId,
     pub regions: Vec<RegionNumber>,
     pub engine: String,
+    pub region_storage_path: String,
```

Comment:
Should we add a migration tool?
```suggestion
    #[serde(default)]
    pub region_storage_path: String,
```

Reply:
it seems the value in `DatanodeTableValue` is not used by others right now? If we are to used it later, then we need to upgrade it first for existing data; or we can do read-update to avoid shutdown maintenance.

Reply:
Starting Datanode (v0.4) requires them.